src/layouts/shared/NetworkErrorBoundary/NetworkErrorBoundary.jsx

```diff
@@ -129,23 +129,32 @@ function ResetHandler({ reset }) {
     }
   }, [history, queryClient, reset])
 
+  const handleSignOut = async () => {
+    await fetch(`${config.API_URL}/logout`, {
+      method: 'POST',
+      credentials: 'include',
+    })
+    history.replace('/login')
+  }
```

Comment:
Hrrm, can we also clear the query client, and reset the error boundary state? Just so that when the user is logged out everything is back into a clean state, and they can start anew.

Reply:
Sure, updated in https://github.com/codecov/gazebo/pull/3119/commits/432553e44ff3f78fa670a97322e0428c0a2ed36b